forChanna.scd

```diff
@@ -2,36 +2,36 @@
     {
         s.options.memSize = 128 * 1024;
         s.options.sampleRate = 48000;
-	// ~libPath = "~/Documents/SC/2019/channa/forChannaHorwitz/lib/".asAbsolutePath; // change as needed
-	    ~libPath = "~/forChannaHorwitz/lib/".asAbsolutePath;
-        ~piecePath = ~libPath ++ "pieces/";
+	~libPath = "~/Documents/SC/2019/channa/forChannaHorwitz/lib/".asAbsolutePath; // change as needed
+
+        ~piecePath = ~libPath ++ "views/players/";
         ~pieces = ();
 
         PathName.new(~piecePath).files.collect({|f|
             f.fullPath.load
         }).do({|p|
             p.init;
-            ~pieces.add(p.title -> p)
+		    ~pieces.add(p.title -> p)
         });
 
-        ~playlist = (~libPath ++ "pieceList.scd").load;
-        ~playlist.create(~pieces, [
-            \channaJar,
+        ~pieceList = (~libPath ++ "/helpers/pieceList.scd").load;
```

Comment:
Can you tell me more about why we'd go back to ```~pieceList``` at this point?

Reply:
ahhh, sorry, this was just a mistake!! Thought I got these all out. I'll fix. playList rules!!

Reply:
No worries; ultimately your call and not trying to be pedantic about it, but I think it should be `~playlist` rather than `~playList`, as it's an instance of something colloquially called a "playlist", not e.g. a function that plays a List.